ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/routing/Route.kt

```diff
@@ -133,3 +133,19 @@ public open class Route(
         }
     }
 }
+
+/**
+ * Return list of endpoints under this route.
```

Comment:
Maybe `Return list of endpoints with handlers under this route.`?

Reply:
yes, done